backend/src/LibCloud/SqlCompiler.fs

```diff
@@ -258,12 +258,21 @@ let (|Fn|_|) (mName : string) (fName : string) (v : int) (expr : Expr) =
 type CompiledSqlQuery =
   { sql : string; vars : List<string * SqlValue>; actualType : TypeReference }
 
+
+let sqlOpForPackageFunction (fnName : FnName.Package) : SqlSpec =
```

Comment:
I think this turned out well!

Reply:
Yes seems better. Thanks for the improvement suggestions!